### ISSUE TITLE
Make the synced lyrics bigger.

### ISSUE THREAD

**Issue by ALKOALSHAN** [vfsfitvnm/ViMusic#441]
### Feature description

It would be really cool and kind of aesthetic if the synced lyrics were bigger so it would be easy read without any struggle. Like the `Spotify Synced Lyrics` or the `Apple Music synced lyrics,` they make it very fluid and smooth. So I recommend tweaking the synced lyrics, I appreciate your work, thanks!

### Why do you want this feature?

> Aesthetically pleasing and appealing 

> Easy to read

**Comment by Abrahamhobandoo** [comment]
Like a full view for the lyrics

**Comment by ALKOALSHAN** [comment]
> Like a full view for the lyrics 

Exactly

**Comment by vfsfitvnm** [comment]
Yes, that would be cool indeed - and it will be done sooner or later

**Comment by TanishMoral11** [comment]
Hey , @ALKOALSHAN Please Merge my code in it , 
i already solved issue #441
